FinSearchUnified/Tests/BusinessLogic/Models/FindologicArticleModelTest.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+use FinSearchUnified\BusinessLogic\FindologicArticleFactory;
+use FinSearchUnified\BusinessLogic\Models\FindologicArticleModel;
+use FinSearchUnified\Tests\Helper\Utility;
+use Shopware\Components\Api\Manager;
+use Shopware\Components\Api\Resource\Article;
+use Shopware\Components\Test\Plugin\TestCase;
+use Shopware\Models\Category\Category;
+
+class FindologicArticleModelTest extends TestCase
+{
+    /** @var Manager */
+    private $manager;
+
+    /** @var FindologicArticleFactory */
+    private $articleFactory;
+
+    protected function setUp()
+    {
+        parent::setUp();
+        $this->manager = new Manager();
+        $this->articleFactory = new FindologicArticleFactory();
+    }
+
+    /**
+     * Method to create test products for the export
+     *
+     * @param array $testProductConfiguration
+     *
+     * @return \Shopware\Models\Article\Article|null
+     */
+    private function createTestProduct($testProductConfiguration)
```

Comment:
Add the type hint here.

Reply:
Done!

Reply:
Ehm, I don't see a change?

Reply:
@ihanli i missunderstood this, thought of the type hints in the docblock. Should be done now.